pyamg/multilevel.py

```diff
@@ -380,6 +380,11 @@ def solve(self, b, x0=None, tol=1e-5, maxiter=100, cycle='V', accel=None,
 
         if accel is not None:
 
+            # Check for symmetric smoothing scheme when using CG
+            if (accel is 'cg') and (self.symmetric_smoothing == False):
+                warn('CG requires SPD matrix. Non-symmetric smoothing scheme '\
+                     'may significantly increase convergence factors.')
```

Comment:
I think a warning like this would be a little more descriptive, telling the user exactly what was detected.
"Incompatible non-symmetric multigrid preconditioner detected, due to presmoother/postsmoother combination.  CG requires SPD preconditioner, not just SPD matrix."